src/Cli/Commands/ConfigureOptions.cs

```diff
@@ -108,6 +122,24 @@ public ConfigureOptions(
         [Option("runtime.cache.ttl-seconds", Required = false, HelpText = "Customize the DAB cache's global default time to live in seconds. Default: 5 seconds (Integer).")]
         public int? RuntimeCacheTTL { get; }
 
+        [Option("runtime.host.mode", Required = false, HelpText = "Set the host running mode of DAB in Development or Production. Default: Development.")]
+        public HostMode? RuntimeHostMode { get; }
+
+        [Option("runtime.host.cors.origins", Required = false, HelpText = "Overwrite Allowed Origins in CORS. Default: [] (Array of strings).")]
```

Comment:
can you add the delimiter we can use to separate origins?

Reply:
"Comma separated array of strings"

Reply:
Its space-separated array of strings. Added